lib/query-interface.js

```diff
@@ -637,8 +637,8 @@ class QueryInterface {
     );
   }
 
-  increment(instance, tableName, values, identifier, options) {
-    const sql = this.QueryGenerator.incrementQuery(tableName, values, identifier, options.attributes);
+  incrementOrDecrement(instance, tableName, values, identifier, options) {
```

Comment:
`Model.decrement` will call `Model.increment`  with `{increment: false}` as its is doing already, at last step we use `increment` to choose between `incrementQuery` or `decrementQuery`.

We will keep `incrementOrDecrementQuery` private, may be rename it to `arithmeticQuery` as well